home/screens/RedesignedSnacksListScreen.tsx/SnackList.tsx

```diff
@@ -0,0 +1,104 @@
+import { Divider, View } from 'expo-dev-client-components';
+import * as React from 'react';
+import { ActivityIndicator, FlatList, View as RNView } from 'react-native';
+import InfiniteScrollView from 'react-native-infinite-scroll-view';
+
+import { RedesignedSnacksListItem } from '../../components/RedesignedSnacksListItem';
+import { CommonSnackDataFragment } from '../../graphql/types';
+
+type Props = {
+  data: CommonSnackDataFragment[];
+  loadMoreAsync: () => Promise<any>;
+};
+
+export function SnackListView(props: Props) {
+  const [isReady, setReady] = React.useState(false);
+
+  React.useEffect(() => {
+    const _readyTimer = setTimeout(() => {
```

Comment:
this seems like something that should be explained by a comment - im not sure what its for other than delaying initial render?

Reply:
I think that's the only purpose. Most of the logic in the redesign is verbatim what existed before just with a new skin, so I wasn't the creator of this code.